### PR TITLE
Implement crosstalk flux matrix fit

### PR DESCRIPTION
**Pull request by andrea-pasquale**
This PR will expand features added in #683 to compute the full crosstalk matrix.

- [x] add table
- [x] fix update
- [x] run resonator flux

Checklist:
- [x] Reviewers confirm new code works as expected.
- [x] Tests are passing.
- [ ] Coverage does not decrease.
- [x] Documentation is updated.
- [x] Compatibility with Qibo modules (Please edit this section if the current pull request is not compatible with the following branches).
    - [x] Qibo: `master`
    - [x] Qibolab: `main`
    - [x] Qibolab_platforms_qrc: `main`

### CODE REVIEW

**Comment by andrea-pasquale** [comment]
Preliminary results on the fit on platinum chip: http://login.qrccluster.com:9000/Z5eKWqZTSzGukFlkCrO8BQ==

File: src/qibocal/protocols/characterization/flux_dependence/qubit_crosstalk.py [hunk 1]
```diff
@@ -68,14 +116,23 @@ def _acquisition(
...
     sequence = PulseSequence()
     ro_pulses = {}
     qd_pulses = {}
+    sweetspots = {}
+    d = {}
+    voltage = {}
+    matrix_element = {}
+    qubit_frequency = {}
     for qubit in targets:
+        sweetspots[qubit] = voltage[qubit] = platform.qubits[qubit].sweetspot
```
[the remark this answers is not in the document]
Comment (andrea-pasquale) [reply src/qibocal/protocols/characterization/flux_dependence/qubit_crosstalk.py:129]: Currently this is what qibolab is doing

**Comment by andrea-pasquale** [comment]
Some results on hardware from this week:

http://login.qrccluster.com:9000/6F4XrE8NQgqFRQjJ6FxprA==
http://login.qrccluster.com:9000/K1IRQq-6TvmZKsSP0aMEYQ==

**Review by igres26 — COMMENTED** [review]
I am trying to run it with this runcard:

  - id: qubit crosstalk
    priority: 0
    # main: t1
    operation: qubit_crosstalk
    parameters:
      flux_qubits: [0, 2, 3]
      drive_duration: 4000
      freq_width: 40_000_000
      freq_step: 200_000
      bias_width: 0.20 #0.1
      bias_step:  0.02 # 0.001
      drive_amplitude: 0.002
      nshots: 5000
      relaxation_time: 2000

And I get this error message:

Traceback (most recent call last):
  File "/nfs/users/sergi.ramos/qiboenv/bin/qq", line 8, in <module>
    sys.exit(command())
  File "/nfs/users/sergi.ramos/qiboenv/lib/python3.10/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
  File "/nfs/users/sergi.ramos/qiboenv/lib/python3.10/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
  File "/nfs/users/sergi.ramos/qiboenv/lib/python3.10/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/nfs/users/sergi.ramos/qiboenv/lib/python3.10/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/nfs/users/sergi.ramos/qiboenv/lib/python3.10/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
  File "/nfs/users/sergi.ramos/qibocal/src/qibocal/cli/_base.py", line 72, in auto
    autocalibrate(runcard, folder, force, update)
  File "/nfs/users/sergi.ramos/qibocal/src/qibocal/cli/autocalibration.py", line 56, in autocalibrate
    for _ in executor.run(mode=ExecutionMode.autocalibration):
  File "/nfs/users/sergi.ramos/qibocal/src/qibocal/auto/execute.py", line 69, in run
    completed = task.run(
  File "/nfs/users/sergi.ramos/qibocal/src/qibocal/auto/task.py", line 108, in run
    completed.data, completed.data_time = operation.acquisition(
  File "/nfs/users/sergi.ramos/qibocal/src/qibocal/auto/operation.py", line 41, in wrapper
    out = func(*args, **kwds)
  File "/nfs/users/sergi.ramos/qibocal/src/qibocal/protocols/characterization/flux_dependence/qubit_crosstalk.py", line 135, in _acquisition
    matrix_element[qubit] = platform.qubits[qubit].crosstalk_matrix[qubit]
KeyError: 0

Am I doing something wrong? Same runcard as I ran it before.

**Comment by andrea-pasquale** [comment]
When you do the crosstalk now the code expects that you should have compute the diagonal fit before. I've relaxed this option in the latest commit.

**Review by igres26 — APPROVED** [review]

**Pull request merged**